prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2735,6 +2735,11 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
+  # {job_prefix}-ironic-image-build-test
+  - name: metal3-ironic-image-build-test
+    agent: jenkins
+    always_run: false
+    optional: false
```

Comment:
If we make this required as it is proposed here, do we want to leave both e2e jobs on too?

Reply:
which both e2e

Reply:
hmm the changes I made on the pipeline does not support the old one anyway

Reply:
I mean ubuntu-e2e-main and centos-e2e-main which are the required tests right now for Ironic-image.